Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM nginx:alpine
 COPY ./packages/core/dist /usr/share/nginx/html
 COPY ./default.conf /etc/nginx/conf.d/default.conf
+RUN chown -R nginx:nginx /usr/share/nginx/html
```

Comment:
EOL please

Reply:
in docker container we are telling nginx process to run onbehalf of `nginx` linux user. but source code directory `/usr/share/nginx/html` used to had `root` user permission. thus source code filesystem permission has been changed to `nginx` user to avoid `root` user privileges